pkg/deploy/generator/resources_rp.go

```diff
@@ -622,14 +630,23 @@ ExecStart=/usr/bin/docker run \
   -e AZURE_ARM_CLIENT_ID \
   -e AZURE_FP_CLIENT_ID \
   -e BILLING_E2E_STORAGE_ACCOUNT_ID \
+  -e CLUSTER_MDSD_ACCOUNT \
   -e CLUSTER_MDSD_CONFIG_VERSION \
+  -e CLUSTER_MDSD_NAMESPACE \
   -e DATABASE_ACCOUNT_NAME \
   -e DOMAIN_NAME \
   -e KEYVAULT_PREFIX \
   -e MDM_ACCOUNT \
   -e MDM_NAMESPACE \
   -e MDSD_ENVIRONMENT \
   -e RP_FEATURES \
+  -e AZURE_EV2 \
+  -e AZURE_ENVIRONMENT \
+  -e AZURE_SUBSCRIPTION_ID \
+  -e AZURE_TENANT_ID \
+  -e LOCATION \
+  -e RESOURCEGROUP \
+  -e HOSTNAME_OVERRIDE \
```

Comment:
This is an unrelated change, is it an outstanding patch CF is carrying?

Reply:
This ultimately came from Rogerio's branch when testing e2e variables (https://github.com/rogbas/ARO-RP/blob/d292f20ce01673b6a9743aa2ddfbb2981e7ef862/pkg/deploy/generator/resources_rp.go#L633)

This is not related to this PR, however, we need to reconvene on all the E2E changes

Reply:
This block has been removed and newly generated files have been added